default/python/universe_generation/universe_tables.py

```diff
@@ -64,14 +64,14 @@
 # none will be forced to have at least one satellite.
 STAR_TYPE_MOD_TO_PLANET_SIZE_DIST = {
     #                           none  tiny  small  medium  large  huge  asteroids  gas giant
-    fo.starType.blue: (0, 0, 0, 0, -5, -10, 0, 0),
-    fo.starType.white: (0, 0, 0, 0, -5, -10, 0, 0),
-    fo.starType.yellow: (0, 0, 0, 0, -5, -10, 0, 0),
-    fo.starType.orange: (0, 0, 0, 0, -5, -10, 0, 0),
-    fo.starType.red: (0, 0, 0, 0, -5, -10, 0, 0),
-    fo.starType.neutron: (5, 0, 0, 0, -5, -10, 20, 0),
+    fo.starType.blue:      (0, 0, 0, 0, -5, -10, 0, 0),
```

Comment:
...

Reply:
this one is missed. correcting. seems to be all of it.